client/app/account/user/credentials/user-credentials.html

```diff
@@ -0,0 +1,103 @@
+<div class="module-useraccount-sections-credentials-container"
+     data-ng-controller="UserAccount.controllers.credentials as $ctrl">
+
+    <div class="page-header">
+        <h1 data-i18n-static="user_credentials_title"></h1>
+    </div>
+
+    <div class="tab-content">
+
+        <div data-ovh-alert="userCredentials"></div>
+
+        <!-- CREDENTIALS EXPLAIN -->
+        <div class="alert alert-info"
+             role="alert">
+            <p>
+                <strong data-i18n-static="user_credentials_info"></strong>
+            </p>
+        </div>
+
+        <!-- CREDENTIALS LIST -->
+        <div class="table-responsive">
+            <table class="table table-hover">
+                <thead>
+                    <tr>
+                        <th></th>
+                        <th data-i18n-static="user_credentials_name" scope="col"></th>
+                        <th data-i18n-static="user_credentials_desc" scope="col"></th>
+                        <th data-i18n-static="user_credentials_creation_date" scope="col"></th>
+                        <th data-i18n-static="user_credentials_expiration_date" scope="col"></th>
+                        <th></th>
+                    </tr>
+                </thead>
+                <tbody data-ng-show="$ctrl.credentialsLoading">
+                    <tr>
+                        <td colspan="8"
+                            class="text-center">
+                            <oui-spinner data-size="s"></oui-spinner>
+                        </td>
+                    </tr>
+                </tbody>
+                <tbody data-ng-show="$ctrl.credentials.length === 0 && !$ctrl.credentialsLoading">
+                    <tr>
+                        <td colspan="8"
+                            class="text-center font-italic"
+                            data-i18n-static="user_credentials_table_empty">
+                        </td>
+                    </tr>
+                </tbody>
+                <tbody data-ng-show="$ctrl.credentials.length > 0 && !$ctrl.credentialsLoading">
+                    <tr data-ng-repeat-start="cred in $ctrl.credentials track by $index">
```

Comment:
As I mentioned to your controller, you can replace your entire table with the `<oui-datagrid>` component and find inspiration in that example:

https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/client/app/dedicated/nas/details/partition/nas-details-partition.html#L50-L81

ref component: http://master.ui-kit.ovh/#!/oui-angular/datagrid